README.md

```diff
@@ -24,30 +26,30 @@ and provides a utility that gets tests up and running quickly
     ```
 
   * writes to `testHelperPath`
-    ```coffee
-    chai = require 'chai'
-    sinon = require 'sinon'
-    chai.use require 'sinon-chai'
+    ```js
+    var chai = require('chai');
+    var sinon = require('sinon');
+    chai.use(require('sinon-chai'));
 
-    global.expect = chai.expect
-    global.sinon = sinon
+    global.chai = chai;
```

Comment:
I've seen us use it fo  other helper files which add custom matchers, but that can also be accomplished by another call to `chai.use` in this file. I'll remove it.